### DISCUSSION TITLE
How to make 'Try it' work with Laravel scantum

### DISCUSSION THREAD

**Discussion by rabol** [dedoc/scramble#212] [Q&A] [answered]
Hi

how can I make the 'Try it' work in  my app using Laravel scantum?

e.g. when I try to login I get an error
```
"message": "CSRF token mismatch.",
```

I know that the login ins working very well using Postman

**Comment by romalytvynenko** [comment]
Hey @rabol 

Can you please elaborate of how your API is set up and how exactly you interact with it using Postman?

I don't fully understand if you're using tokens, or somehow you use session data in Postman.

Thanks!

**Reply by rabol** [reply]
It's very common to use Laravel scantum and when one do so, and try to do any post request from the documentation pages one will the the error.

on the other hand, if you use Postman to test your API, everything works correct, so in my case, I know that the API is working, but it does not work from the 'try me' as there is no CSRF

**Comment by ChrisSantiago82** [comment]
I have the same issue with scantum.
Of course, with scantum on an api, we wouldn't use a CSRF token.

**Comment by ChrisSantiago82** [comment]
Removing the "\Laravel\Sanctum\Http\Middleware\EnsureFrontendRequestsAreStateful::class," from Kernel solved my issue.

**Reply by layerok** [reply]
Yeah, It will work for token-based authentication, but it won't work for cookie-based authentication. You have simply disabled the ability to authenticate users via session cookies.

**Comment by romalytvynenko** [comment, marked as answer]
@rabol this should be fixed in https://github.com/dedoc/scramble/releases/tag/v0.8.5, please check it out. Feel free to re-open discussion if not working.
